src/time_evolution/mcsolve.jl

```diff
@@ -242,29 +268,52 @@ end
         output_func::Function=_mcsolve_output_func,
         kwargs...)
 
-Generates the ODEProblem for an ensemble of trajectories of the Monte Carlo wave function
-time evolution of an open quantum system.
+Generates the ODEProblem for an ensemble of trajectories of the Monte Carlo wave function time evolution of an open quantum system.
```

Comment:
I would write “Generates the `EnsembleProblem` of `ODEProblem`s for the ensemble of trajectories of the Monte Carlo time evolution …”

Reply:
Done~